code/modules/customitems/item_defines.dm

```diff
@@ -830,3 +830,13 @@
 	new /obj/item/weapon/reagent_containers/food/drinks/drinkingglass/shotglass(src)
 	new /obj/item/weapon/reagent_containers/food/drinks/drinkingglass/shotglass(src)
 	new /obj/item/weapon/reagent_containers/food/drinks/bottle/fluff/moonshine(src)
+
+
+
+/obj/item/clothing/accessory/necklace/locket/fethasnecklace //Fethas: Sefra'neem
+	name = "Orange gemmed locket"
+	desc = "a locket with a orange gem set on the front, the picture inside seems to be of a Tajaran."
```

Comment:
Pedantry: all the other desc's started with the first letter of the first word capitalised. This should remain consistent.